pkg/imgpkg/imageset/tar_image_set.go

```diff
@@ -27,21 +29,64 @@ func NewTarImageSet(imageSet ImageSet, concurrency int, ui goui.UI) TarImageSet
 }
 
 // Export Creates a Tar with the provided Images
-func (i TarImageSet) Export(foundImages *UnprocessedImageRefs, outputPath string, registry registry.ImagesReaderWriter, imageLayerWriterCheck imagetar.ImageLayerWriterFilter) (*imagedesc.ImageRefDescriptors, error) {
+func (i TarImageSet) Export(foundImages *UnprocessedImageRefs, outputPath string, registry registry.ImagesReaderWriter, imageLayerWriterCheck imagetar.ImageLayerWriterFilter, resume bool) (d *imagedesc.ImageRefDescriptors, err error) {
 	ids, err := i.imageSet.Export(foundImages, registry)
 	if err != nil {
 		return nil, err
 	}
 
-	outputFile, err := os.Create(outputPath)
+	var outputFile *os.File
+	var layers []v1.Layer
+	var tmpFile *os.File
```

Comment:
it's not clear by the name how tmpFile is used (this deserves a comment). it seems like we do quite a bit of juggling with it.